File: pyg_parent/pinyougou_content_interface/src/main/java/com/pinyougou/content/service/PromotionService.java
package com.pinyougou.content.service;

import com.pyg_pojo.entity.PageResult;
import com.pyg_pojo.pojo.TbPromotion;

import java.util.List;

public interface PromotionService {

    //修改活动信息
    public abstract void update(TbPromotion promotion);

    //删除活动信息
    public abstract void delete(String[] ids);

    //条件分页查询
    public abstract PageResult findPage(Integer pageNum, Integer pageSize, TbPromotion tbPromotion);

    //添加活動信息
    public abstract void add(TbPromotion promotion);

    //按照id查询
    public abstract TbPromotion findOne(String id);

    //根据状态查询
    public abstract List<TbPromotion> findStatus();
}

File: pyg_parent/pyg_manager_web/src/main/webapp/js/service/promotionService.js
app.service('promotionService',function ($http) {

    //查询实体
    this.findOne=function(id){
        return $http.get('../promotionController/findOne.do?id='+id);
    }
    //增加 
    this.add=function(entity){
        return  $http.post('../promotionController/add.do',entity );
    }
    //修改 
    this.update=function(entity){
        return  $http.post('../promotionController/update.do',entity );
    }
    //删除
    this.dele=function(ids){
        return $http.get('../promotionController/delete.do?ids='+ids);
    }
    //搜索
    this.search=function(page,rows,searchEntity){
        return $http.post('../promotionController/search.do?pageNum='+page+"&pageSize="+rows, searchEntity);
    }

})

